src/SFA.DAS.ApprenticeCommitments.Database/PostDeployment/PostDeployment.sql

```diff
@@ -0,0 +1,27 @@
+﻿/*
+Post-Deployment Script Template							
+--------------------------------------------------------------------------------------
+ This file contains SQL statements that will be appended to the build script.		
+ Use SQLCMD syntax to include a file in the post-deployment script.			
+ Example:      :r .\myfile.sql								
+ Use SQLCMD syntax to reference a variable in the post-deployment script.		
+ Example:      :setvar TableName MyTable							
+               SELECT * FROM [$(TableName)]					
+--------------------------------------------------------------------------------------
+*/
+
+
+/* Developer Role Access */
+
+IF DATABASE_PRINCIPAL_ID('Developer') IS NULL
+BEGIN
+    CREATE ROLE [Developer]
+END
+
+GRANT SELECT ON ApprenticeDashboardView TO Developer
+
+GRANT SELECT ON ApprenticeshipDashboardView TO Developer
+
+GRANT SELECT ON CommitmentstatementDashboardView TO Developer
+
+GRANT SELECT ON RegistrationDashboardView TO Developer

```

Comment:
Now I may be wrong here, as SQL changes from version to version. But we used to have to grant access to the underlying tables here as well as the view (hopefully, this requirement no longer exists), but worth double checking